### ISSUE TITLE
Preview changes not appearing in lookbook.

### ISSUE THREAD

**Issue by DRBragg** [allmarkedup/lookbook#93]
I'm running into a strange issue where changes I make to any preview files are not appearing in the Lookbook web UI. I can view the previews through rails normally but they're not in lookbook. 

With a little bit of digging I found that if I manually run `Lookbook::Engine.parser.parse` before starting my server the changes do appear. I _think_ this is because we have `config.lookbook.listen` set to `false` and in `lib/lookbook/engine.rb` it looks like the initial `parse` call:
```ruby
if config.lookbook.runtime_parsing
  parser.parse
else
  unless File.exist?(config.lookbook.parser_registry_path)
    Lookbook.logger.warn "
      Runtime parsing is disabled but no registry file has been found.
      Did you run `rake lookbook:preparse` before starting the app?
      Expected to find registry file at #{config.lookbook.parser_registry_path}
    "
  end
end
```
is _inside_ the `if config.lookbook.listen` conditional of the `config.after_initialize` block.

In my local copy of Lookbook I tried moving the `if config.lookbook.runtime_parsing` outside of the `if config.lookbook.listen` conditional and everything started working as (I) expected.

I'm not sure if this is a bug or intentional. If it's a bug I'm happy to open a PR with a fix.  If it's intentional then there need to be a `parser.parse` call somewhere after the `if config.lookbook.listen` conditional to ensure that the yard registry is updated correctly.  

Let me know what you think.

**Comment by allmarkedup** [comment]
Hey @DRBragg many thanks for this - it's definitely a bug! It must have crept in in the last release. 

I'll get it fixed now and get a new release out shortly 👍

**Comment by DRBragg** [comment]
Awesome! Thank you for all your hard work, @allmarkedup! Once you have this fixed I'll merge it into my fork (We need to support Ruby 2.5) and then I'll be able to release Lookbook to the whole team. Very excited!

**Comment by allmarkedup** [comment]
@DRBragg my pleasure! I've just released v0.8.2 which should fix the issue.

Thanks again for the bug report and for digging into it and just let me know if you run into any other issues.
